src/main/java/seedu/address/model/entitylist/ParticipantList.java

```diff
@@ -170,4 +178,16 @@ public static Id generateId() {
     public static void setLastUsedId(int number) {
         lastUsedId = number;
     }
+
+    /**
+     * Provides a deep copy of the ParticipantList
+     * @return Deep copy of ParticipantList
+     */
+    public ParticipantList copy() throws AlfredException {
```

Comment:
Maybe you would want to throw a more specific exception(hahah I don't know I think its good practise)? It took me a while to find out that it is actually AlfredModelException

Reply:
Yup you are right! I'll make the change now. Can't believe I missed this haha